async.js

```diff
@@ -0,0 +1,23 @@
+const readline = require('readline')
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+async function main() {
+  const name = await askQuestion(readlineInterface, "What is your name? ")
+  const job = await askQuestion(readlineInterface, "What is your job? ")
+  const age = await askQuestion(readlineInterface, "How old are you? ")
+  
+  console.log(`Hello ${name}. You are a ${age} year old ${job}.`)
+  readlineInterface.close()
+}
+main()
+
+function askQuestion(readlineInterface, question) {
+  return new Promise(resolve => {
```

Comment:
**GPT COMMENT**: 
 - The 'askQuestion' function is using a Promise constructor. It would be better to use a more modern async/await syntax to make the code cleaner and easier to understand.